docs/explanations/pro_token_and_machine_usage.rst

```diff
@@ -0,0 +1,71 @@
+.. _pro_token_and_machine_usage:
+
+Machine counts and token usage for Ubuntu Pro
+*********************************************
+
+Ubuntu Pro tokens are used to authenticate your machine and enable access to the services you are entitled to. tokens are unique and specific to your account, so you should treat them as a secret (as you would with a password).
+
+Since tokens are attached to your account and not your machine, you can re-use tokens, even multiple times per day. In this page, we'll discuss how tokens work and how machines are counted against your subscription.
+
+Pro tokens
+----------
+
+- Obtaining tokens:
+    After subscribing to Ubuntu Pro, you can access your tokens through the `Ubuntu Pro Dashboard <Pro_dashboard_>`_. Each token can be used to attach multiple machines depending on the subscription.
+
+- Attaching tokens:
+    Command: Use ``sudo pro attach`` to connect a machine to your Ubuntu Pro subscription.
+
+- Detaching tokens:
+    Command: Use ``sudo pro detach`` to remove a machine from the subscription. Once a machine is detached, the token can be reused to attach another machine. If you are attached on a VM you intend to delete, the token does not need to be detached before the VM is destroyed.
+
+For further information regarding pro commands, refer to our :ref:`Getting started-with Ubuntu Pro client tutorial<tutorial-commands>`.
+
+Monitoring active machines:
+---------------------------
+
+Active machines are counted based on the number of machines attached to your subscription at a given time.
+
+- Machine Limits: 
+    Each subscription type has a limit on the number of machines it can cover. For personal subscriptions, this limit is 5 physical machines with unlimited VMs on these machines. For enterprise subscriptions, the limit is based on the number of physical hosts and VMs. 
+    For extensive use beyond 5 machines, consider an enterprise subscription.
+
+- Active Machine Tracking: 
+    The Ubuntu Pro dashboard and tools like `Landscape <Landscape_>`_ help track the number of active machines, ensuring compliance with subscription limits.
+
+- Unused Licenses and Active Users:
+    The Ubuntu Pro dashboard provides a comprehensive view of all active and attached machines. It shows the number of active machines and helps track unused Landscape licenses, ensuring that you remain within the subscription limits.
+
+- Monitoring with Landscape:
+    Setup: Install the Landscape client and connect it to the Landscape server. For instructions on how to install the Landscape client, refer to the `Landscape Installation Guide <Landscape_install_>`_.
+
+    Audit Machines: Use Landscape’s remote script execution capability to run audit scripts, such as ``uastatus.sh``, to gather detailed information about your Ubuntu Pro subscription across all machines. This information can be exported into a CSV report for further analysis.
```

Comment:
I think we shuold limit this doc to the Pro subscription model's relation ship to tokens and machine activity, so all of the pieces that talk about using landscape to monitor things can be removed IMO
```suggestion
```

Reply:
Do we want to point towards Landscape for monitoring enterprise machines or would that be diverting from the main topic of the doc?

Reply:
We might need to add Landscape stuff back in later (once we start moving towards adding this also to the Pro Handbook) but I think that can be done as future work if necessary.